Source/RageCommunity.Library/Wrappers/NativeLocalizationWrappers.cs

```diff
@@ -6,6 +6,13 @@ namespace RageCommunity.Library.Wrappers
 {
     public static partial class NativeWrappers
     {
-        
+        /// <summary>
+        /// Get the current game language.
+        /// </summary>
+        /// <returns>A number from 0-12, representing a language. 0 = american. For more languages see <a href="https://alloc8or.re/gta5/nativedb/?n=0x2BDD44CC428A7EAE">Grant Theft Auto V native DB</a>.</returns>
+        public static int GetGameLanguage()
```

Comment:
The name of this native on NativeDB is GET_CURRENT_LANGUAGE
```suggestion
        public static int GetCurrentLanguage()
```